lib/error-message-factory-no-op.js

```diff
@@ -0,0 +1,5 @@
+const errorMessageFactory = (errorCode, extras) => {
+  return `An error was thrown by workbox with error code: '${errorCode}' and extras: '${JSON.stringify(extras)}'.`;
```

Comment:
How about:

```js
const errorMessageFactory = (code, context) => {
  return `Workbox encountered an error:\n${JSON.stringify({code, context}, null, 2)}`;
};
```

If `context` isn't set, that will leave out the `context` key from the JSON output.